diff --git a/node_modules/fastembed/lib/cjs/fastembed.js b/node_modules/fastembed/lib/cjs/fastembed.js
index 16ff4e4..8f2a882 100644
--- a/node_modules/fastembed/lib/cjs/fastembed.js
+++ b/node_modules/fastembed/lib/cjs/fastembed.js
@@ -33,7 +33,7 @@ const https_1 = __importDefault(require("https"));
 const ort = __importStar(require("onnxruntime-node"));
 const path_1 = __importDefault(require("path"));
 const progress_1 = __importDefault(require("progress"));
-const tar_1 = __importDefault(require("tar"));
+const tar_1 = { default: require("tar") };
 const hub_1 = require("@huggingface/hub");
 var ExecutionProvider;
 (function (ExecutionProvider) {
diff --git a/node_modules/fastembed/lib/esm/fastembed.js b/node_modules/fastembed/lib/esm/fastembed.js
index 620362e..3e1ecd1 100644
--- a/node_modules/fastembed/lib/esm/fastembed.js
+++ b/node_modules/fastembed/lib/esm/fastembed.js
@@ -4,7 +4,7 @@ import https from "https";
 import * as ort from "onnxruntime-node";
 import path from "path";
 import Progress from "progress";
-import tar from "tar";
+import * as tar from "tar";
 import { downloadFileToCacheDir } from "@huggingface/hub";
 export var ExecutionProvider;
 (function (ExecutionProvider) {
